src/common/components/home/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+                        community-governed NFT platform on Solana.
+                    </div>
+                    <SocialLinks/>
+                </div>
+                <div className="grid lg:grid-cols-4 sm:grid-cols-2 gap-12">
+                    <div className="justify-start">
+                        <span className="text-white text-base font-medium">About</span>
+                        <ul className="mt-4 space-y-2">
+                            <li><FooterLink href="/about">About Holaplex</FooterLink></li>
+                            <li><FooterLink newTab href="https://docs.google.com/document/d/1jskpoCdDm7DU2IbeXwRhhl5LGiNhonAx2HsmfJlDsEs">Terms of Service</FooterLink></li>
+                            <li><FooterLink newTab href="https://docs.google.com/document/d/12uQU7LbLUd0bY7Nz13-F9cua5Wk8mnRNBlyDzF6gRmo">Privacy Policy</FooterLink></li>
+                        </ul>
+                    </div>
+                    <div className="justify-start">
+                        <span className="text-white text-base font-medium">Create</span>
+                        <ul className="mt-4 space-y-2">
+                            <li><FooterLink href="/nfts/new">Mint NFTs</FooterLink></li>
+                            <li><FooterLink href="/marketplace/new"><TaggedNew>Marketplaces</TaggedNew></FooterLink></li>
+                            <li><FooterLink href="/storefront/new">Set up a storefront</FooterLink></li>
+                        </ul>
+                    </div>
+                    <div className="justify-start">
+                        <span className="text-white text-base font-medium">Community</span>
+                        <ul className="mt-4 space-y-2">
+                            <li><FooterLink newTab href="https://github.com/holaplex">GitHub Repos</FooterLink></li>
+                            <li><FooterLink newTab href="https://airtable.com/shrXoZ9qU9g4oeM81">Hola Collective</FooterLink></li>
+                            <li><FooterLink href="#">HolaListings</FooterLink></li>
+                        </ul>
+                    </div>
+                    <div className="justify-start">
+                        <span className="text-white text-base font-medium">Help</span>
+                        <ul className="mt-4 space-y-2">
+                            <li><FooterLink newTab href="https://holaplex-support.zendesk.com/hc/en-us/sections/4407417107475-FAQ">FAQs</FooterLink></li>
+                            <li><FooterLink newTab href="https://holaplex-support.zendesk.com/hc/en-us/requests/new">Support</FooterLink></li>
+                            <li><FooterLink newTab href="https://twitter.com/holastatus">HolaStatus</FooterLink></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </footer>
+    );
+}
+
+
+interface LinkProps {
+    href: string;
+    newTab?: boolean;
+}
+
+const FooterLink: FC<LinkProps> = props => {
```

Comment:
does my FooterLink class makes sense, as it essentially just wraps up some tailwind classes with no useful additional logic, which seems like a tailwind no-no but also makes my life easier

Reply:
@austinmilt this is good you are just setting the default for children elements could be overridden it on each child element